{"name": "test_run", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0", "trace": "self = <test_case.Test_all01 object at 0x0395BB08>\n\n    @allure.step(title=\"run就是一个正常的方法.\")\n    def test_run(self):\n        allure.attach(\"自定义描述1\", \"描述内容，自定义\")\n        print(\"我要运行\")\n>       assert 0\nE       assert 0\n\njenkins_demo\\scripts\\test_case.py:15: AssertionError"}, "steps": [{"name": "run就是一个正常的方法.", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0\n", "trace": "  File \"d:\\python\\jenkins_project\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"D:\\python\\jenkins_project\\jenkins_demo\\scripts\\test_case.py\", line 15, in test_run\n    assert 0\n"}, "attachments": [{"name": "描述内容，自定义", "source": "9a53d851-bfa9-4650-ac06-75f4683857db-attachment.attach"}], "start": 1626518003458, "stop": 1626518003459}], "start": 1626518003458, "stop": 1626518003461, "uuid": "97d25ebd-fbae-48ec-af34-ca5d3a6a175c", "historyId": "733d47e25a2559bb0babb7704ec658d5", "testCaseId": "e7d4edf814676dd5e95bc53f38f1be1c", "fullName": "jenkins_demo.scripts.test_case.Test_all01#test_run", "labels": [{"name": "feature", "value": "Test_all01"}, {"name": "parentSuite", "value": "jenkins_demo.scripts"}, {"name": "suite", "value": "test_case"}, {"name": "subSuite", "value": "Test_all01"}, {"name": "host", "value": "BJ3184"}, {"name": "thread", "value": "9244-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "jenkins_demo.scripts.test_case"}]}